src/meson.build

```diff
@@ -59,7 +59,10 @@ endif
 
 if get_option('opengl')
 	cflags += ['-DCONFIG_OPENGL', '-DGL_GLEXT_PROTOTYPES']
```

Comment:
do we still need `-DGL_GLEXT_PROTOTYPES`?

Reply:
good catch